src/ui/Windows/ThisMfcApp.cpp

```diff
@@ -1474,39 +1471,42 @@ void ThisMfcApp::GetLanguageFiles()
     // Create LCID
     LCID lcid = MAKELCID(FoundResLangID, SORT_DEFAULT);
 
-    inum = ::GetLocaleInfo(lcid, LOCALE_SNATIVELANGNAME, NULL, 0);
+    inum = ::GetLocaleInfo(lcid, LOCALE_SNATIVELANGNAME, nullptr, 0);
     if (inum > 0) {
+      std::unique_ptr<wchar_t[]> szLanguage_English;
+      std::unique_ptr<wchar_t[]> szCountry_Native;
+      std::unique_ptr<wchar_t[]> szCountry_English;
```

Comment:
Could we similarly reduce the scope of the last two variables to the if block below?

Reply:
> Could we similarly reduce the scope of the last two variables to the if block below?

Unfortunately, they are initialized in the scope starting on line 1484, but then used in the scope starting on line 1520.